pmm/pmm2-api-tests.groovy

```diff
@@ -86,13 +86,13 @@ pipeline {
             steps {
                 sh '''
                     sudo docker run -e PMM_SERVER_URL=\${PMM_URL} pmm-api-tests
+                    sudo docker cp pmm-api-tests:/go/src/github.com/Percona-Lab/pmm-api-tests/pmm-api-tests-junit-report.xml ./pmm-api-tests-junit-report.xml
```

Comment:
this will not be executed every time, so you might end up getting no results.

Reply:
Yes, but we can get report only after tests runs. And in my current implementation, it doesn't work at all, because this script doesn't run :( Check it here: https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-api-tests/detail/pmm2-api-tests/822/pipeline/39